lib/avtp_pipeline/intf_echo/openavb_intf_echo.c

```diff
@@ -209,7 +216,12 @@ bool openavbIntfEchoTxCB(media_q_t *pMediaQ)
 					printf("%s\n\r", (char *)pMediaQItem->pPubData); 
 			}
 
+#if IGB_LAUNCHTIME_ENABLED
+				openavbAvtpTimeAddUSec(&pPvtData->walltime, 125);
```

Comment:
Adding a constant 125 usec means this is fixed for Class A (8000ps).  I believe this should be based on the tx rate specified in the .ini file for the stream.